README.md

```diff
@@ -25,6 +25,10 @@ If you're having an issue with a particular plugin, open an issue in that plugin
 
 There is  a [Homebridge community on Reddit](https://www.reddit.com/r/homebridge/).
 
+Our official Discord community is [here](https://discord.gg/RcV7fa8).
```

Comment:
Would you mind to update the language to something that reflects the purpose of Discord server? It's more of a general HomeKit discussion discord server, instead of a homebridge one.

Reply:
While we do have a dedicated section focused on Homebridge and support for it, I've gone ahead and updated it.